project/Dependencies.scala

```diff
@@ -65,6 +65,7 @@ object Dependencies {
   )
 
   private val sprayServerDependencies = List(
+    "org.webjars" % "swagger-ui" % "2.2.2" % Provided,
```

Comment:
@kshakir how the hell do i reference `swaggerUiVersion` from here? it wasn't even like that before.